mod_upload/forms.py

```diff
@@ -43,6 +43,12 @@ class CommonSampleForm(Form):
     version = SelectField(
         'Version', [DataRequired(message='Version is not selected')],
         coerce=int)
+    report = SelectField('Do you want to report Issue on GitHub?', choices=[
+                         ('n', 'No'), ('y', 'Yes')])
+    IssueTitle = TextAreaField(
+        'Issue Title', [DataRequired(message='Title is not filled in')])
```

Comment:
In templates/upload/process_id.html javascript, I have added in L85-L87, I have set the value to input boxes if checkbox of issue is not set.